### ISSUE TITLE
Multi-Level BinaryTree does not work

### ISSUE THREAD

**Issue by alexporth** [RunestoneInteractive/pythonds#75]
The BinaryTree class does not work if you are creating one with more than two layers, as the children are assigned to the wrong instance variables in the insert methods.

Method insertLeft | Line 30:
t.left = self.leftChild
should be 
t.leftChild = self.leftChild

Method insertRight | Line 41:
t.right = self.rightChild
should be
t.rightChild = self.rightChild

**Comment by owenslo** [comment]
Hello,
Could you share which chapter in the book you found this issue?
-Logan

**Comment by EosEtIris** [comment]
It came up when I was trying the Programming Exercises in chapter 7. When I wrote my own strings to test my implementation, for the problem in exercise 1, I had something along the lines of "(( 4+6 ) -( 10 + 5 )*3)" as the input. This resulted in part of the statement being completely ignored.

When I debugged it in Pycharm I found that the parts of the expression were being stored in variables that were never called. I can send you the exact expression that I used if you want it. I just don't have access to my computer right now.
